layers/+lang/emacs-lisp/extensions.el

```diff
@@ -15,17 +15,59 @@
 
 (defun emacs-lisp/init-emacs-builtin-emacs-lisp ()
 
-  (evil-leader/set-key-for-mode 'emacs-lisp-mode
-    "me$" 'lisp-state-eval-sexp-end-of-line
-    "meb" 'eval-buffer
-    "mec" 'spacemacs/eval-current-form
-    "mee" 'eval-last-sexp
-    "mer" 'eval-region
-    "mef" 'eval-defun
-    "mel" 'lisp-state-eval-sexp-end-of-line
-    "m,"  'lisp-state-toggle-lisp-state
-    "mtb" 'spacemacs/ert-run-tests-buffer
-    "mtq" 'ert)
+  (dolist (mode '(emacs-lisp-mode lisp-interaction-mode))
+    (evil-leader/set-key-for-mode mode
+      "me$" 'lisp-state-eval-sexp-end-of-line
+      "meb" 'eval-buffer
+      "mec" 'spacemacs/eval-current-form
+      "mee" 'eval-last-sexp
+      "mer" 'eval-region
+      "mef" 'eval-defun
+      "mel" 'lisp-state-eval-sexp-end-of-line
+      "m,"  'lisp-state-toggle-lisp-state
+      "mtb" 'spacemacs/ert-run-tests-buffer
+      "mtq" 'ert))
+
+  (if (configuration-layer/layer-usedp 'spacemacs)
```

Comment:
Do it the other way around, add `emacs-builtin-emacs-lisp` to the layer `spacemacs` and di your stuff in a post-init.